packages/components/nodes/prompts/PromptTemplate/PromptTemplate.ts

```diff
@@ -45,7 +45,20 @@ class PromptTemplate_Prompts implements INode {
 
         let promptValues: ICommonObject = {}
         if (promptValuesStr) {
-            promptValues = JSON.parse(promptValuesStr.replace(/\s/g, ''))
+            promptValues = JSON.parse(promptValuesStr)
```

Comment:
need to apply changes to `ChatPromptTemplate` and `FewShotPromptTemplate` and test